### PR TITLE
Xero Projects

### PR DESCRIPTION
**Pull request by troystaylor**
---
When submitting a connector, please check the following conditions for your PR.

- [X] I attest that the connector works and I verified by deploying and testing all the operations.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/Xero Projects/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,582 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Xero Projects/apiDefinition.swagger.json:4]: @troystaylor Can you please provide your connector name as title.

Suggestion : `Xero Projects`

File: independent-publisher-connectors/Xero Projects/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+### Action: Update a task
+Allows you to update a task.
+### Action: Delete a task
+Allows you to delete a task.
+
+## Known Issues and Limitations
+There are no known issues at time of publishing.
+
+## Frequently Asked Questions
+
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Xero Projects/README.md:57]: @troystaylor You can remove this section if there is nothing to mention anything

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by troystaylor** [comment]
sriyen-msft Fixed

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@troystaylor Apologies, I have accidently merged this PR which is not supposed to do. We need to just approve the PR and keep it open to pick our deployment system for deployment. Could you please resubmit the PR for `Xero Projects` and `Xero Payroll - UK` ?? Thank you so much in advance.

**Pull request merged**